torchaudio/functional/functional.py

```diff
@@ -1444,3 +1446,46 @@ def resample(
                                               resampling_method, beta, waveform.device, waveform.dtype)
     resampled = _apply_sinc_resample_kernel(waveform, orig_freq, new_freq, gcd, kernel, width)
     return resampled
+
+
+@torch.jit.unused
+def edit_distance(seq1: Sequence, seq2: Sequence) -> int:
+    """
+    Calculate the word level edit (Levenshtein) distance between two seqences.
+
+    The function computes an edit distance allowing deletion, insertion and
+    substitution. The result is an integer.
+
+    For most applications, the two input sequences should be the same type.
+    If a pair strings are given, the output would be the edit distance between
+    two words (character edit distance); if two lists of strings are given, the
+    output would be the edit distance between sentences (word edit distance).
+    Users may want to normalize the output by the length of the reference
+    sequence.
```

Comment:
```suggestion
    Compute the (Levenshtein) edit distance between two sequences,
    allowing deletion, insertion and substitution. The result is an integer.

    For most applications, the two input sequences should be the same type.
    If two strings are given, the output is the edit distance between the two
    (character edit distance). If two lists of strings are given, the
    output is the edit distance between sentences (word edit distance).
    Users may want to normalize the output by the length of the reference
    sequence.
```

Reply:
nit: misspelled sequence

Reply:
Updated it [here](https://github.com/pytorch/audio/pull/1601/files#diff-cf61df2ded7de3af992d120c6b1530bddb6333ad62e825ce4c9251dafe9afd1dR406)

Reply:
There is more than just the typo to change. I've updated some of the phrasing:
```
    Calculate the (Levenshtein) edit distance between two sequences.

    The function computes an edit distance allowing deletion, insertion and
    substitution. The result is an integer.

    For most applications, the two input sequences should be the same type.
    If two strings are given, the output is the edit distance between
    the two strings (character edit distance). If two lists of strings are given, the
    output is the edit distance between sentences (word edit distance).
    Users may want to normalize the output by the length of the reference
    sequence.
```